src/components/board-settings/tabs/SubjectsTab.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div>
+    <h1 class="headline mb-6 text-center">
+      Przedmioty
+    </h1>
+    <v-card
+      v-for="subject in subjectsList"
+      :key="subject.id"
+      class="mb-3"
+      outlined
+    >
+      <v-row
+        class="align-center pl-4 pr-5 py-2"
+        no-gutters
+      >
+        <v-col cols="auto">
+          <v-sheet
+            :color="subject.color"
+            :height="30"
+            :width="60"
+          />
+        </v-col>
+        <v-col>
+          <v-hover
+            v-slot:default="{ hover }"
+            open-delay="200"
+          >
```

Comment:
The use of v-hover makes it hard/unclear to use on mobile. The input is not accessible via keyboard-only / reader input methods. The text-field may disappear when typing if the user moves their mouse.